script/jenkins/test-install-packages.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -xeuo pipefail
 
-mage -v -debug testPackagesInstall
+make test-packages-install
```

Comment:
we preferred to use mage test-packages it is simple and does not have any no-build stuff